app/controllers/shelters_controller.rb

```diff
@@ -48,6 +48,11 @@ def update
   def show
     @shelter = Shelter.find(params[:id])
     @shelter_reviews = @shelter.shelter_reviews
+    if @shelter_reviews.length >= 1
```

Comment:
Do we need a conditional here? Unless there's something I'm missing, I think it'd work and would be cleaner to just leave it at `@review_average = @shelter_reviews.sum {|review| review.rating} / @shelter_reviews.count` since this code would take into account the number of reviews anyway, even if it's just one review.

Reply:
Also, because this is a calculation, it should probably go in the `ShelterReview` model

Reply:
I had to put the else, because when I had just 1 review, I was getting an error like: "Cannot divide number by 1" or something to that effect.  Maybe there is a better way we can do it though?  I'm definitely down to refactor, I just ran out of gas last night, kept getting bogged down trying to figure out AR stuff.

Reply:
Gotcha - we can refactor the if/else stuff later if we have time then! But I do think we should make sure to follow the MVC conventions with this one since that is a part of the project evaluation, ie. putting this calculation in a model not a controller. Let me know if you think otherwise or if there's a reason specifically to keep it in here.